src/applications/vaos/sass/vaos.scss

```diff
@@ -216,6 +216,10 @@
   }
 }
 
+.vaos-facility-details__day {
+  width: 110px;
```

Comment:
Is the expectation from VSP that we shouldn’t be writing custom CSS? I'm not sure I understand what prompted this comment. There’s not an obvious helper class for this style and you can see the design we're building in the screenshot for this issue, which is not covered in the design system (though it is used in the facility locator).

Reply:
this is new design that will be added to the design system once it's been thoroughly tested and reviewed.

Reply:
We generally try to avoid custom CSS.  I was wondering if something else had been tried first but just didn't work out, like using a [fixed column layout](https://design.va.gov/layout/flexbox-grid#fixed-columns).

Reply:
Ok, gotcha. Narin can better speak to that, but this is another case of trying to adapt some copied code into our app.

I totally agree with reducing custom CSS (we'd be almost entirely avoiding it we didn't have a new calendar component), we just hadn't gotten comments on some of the other custom CSS in past PRs where there are some judgment calls about whether to use the helper classes or not, so I wasn't sure if this was a changed expectation.

Reply:
Will try to restyle it using the fixed column utilities.